internal/kubernetes/reflectionController.go

```diff
@@ -112,6 +112,18 @@ func (p *KubernetesProvider) controlLoop() {
 
 func (p *KubernetesProvider) cleanupNamespace(ns string) error {
 
+	pods, err := p.foreignClient.Client().CoreV1().Pods(ns).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, po := range pods.Items {
+		err = p.foreignClient.Client().CoreV1().Pods(ns).Delete(po.Name, &metav1.DeleteOptions{})
+		if err != nil {
+			klog.Error(err, "cannot delete remote pod %v", po.Name)
```

Comment:
In order to use the escape feature, you should use `klog.errorf` instead of `klog.error`